src/peft/tuners/lora/config.py

```diff
@@ -117,6 +118,25 @@ class LoraConfig(PeftConfig):
             )
         },
     )
+    bits_pattern: Optional[dict] = field(
```

Comment:
Hmm, there are a bunch of new parameters. Maybe it would be better to have a separate `LoftQConfig` class only for the LoftQ-parameters. We could then pass this config to `LoraConfig`. So in practice, it would look something like this for users:

```python
loftq_config = LoftQConfig(bits_pattern=..., loftq_bits=...)
lora_config = LoraConfig(r=16, ..., loftq_config=loftq_config)
```

By default, `loftq_config` on `LoraConfig` would be `None`, but if users pass it, it means that they want to use LoftQ-initialization. How does that sound?

@pacman100 WDYT about that?